JitsiConnection.js

```diff
@@ -25,12 +26,19 @@ function JitsiConnection(appID, token, options) {
 
     this.addEventListener(JitsiConnectionEvents.CONNECTION_FAILED,
         function (errType, msg) {
+            AnalyticsAdapter.sendEvent('connection.failed.' + errType);
             if(errType === JitsiConnectionErrors.OTHER_ERROR &&
                 (msg === "item-not-found" || msg === "host-unknown")) {
                     // FIXME: don't report the error if we are going to reload
                     this._reload();
                 }
         }.bind(this));
+
+    this.addEventListener(JitsiConnectionEvents.CONNECTION_DISCONNECTED,
+        function (msg) {
+            if(msg)
+                AnalyticsAdapter.sendEvent('connection.disconnected.' + msg);
```

Comment:
My concern is that with the current implementation we don't log when msg is falsy (undefined, null, '', etc.). If we don't care about these cases, then we're good :)